dsc_lib/src/configure/context.rs

```diff
@@ -12,6 +12,7 @@ use super::config_doc::{DataType, SecurityContextKind};
 pub struct Context {
     pub execution_type: ExecutionKind,
     pub outputs: HashMap<String, Value>, // this is used by the `reference()` function to retrieve output
+    pub mounted_path: String,
```

Comment:
How did you decide on using a `String`? Curious if `PathBuf` would be useful here.

Reply:
Should probably be PathBuf as appending takes care of dir separator automatically.